go/protocol/keybase1/notify_fs_request.go

```diff
@@ -12,8 +12,13 @@ type FSEditListRequestArg struct {
 	Req FSEditListRequest `codec:"req" json:"req"`
 }
 
+type FSSyncStatusRequestArg struct {
+	Req FSSyncStatusRequest `codec:"req" json:"req"`
+}
+
 type NotifyFSRequestInterface interface {
 	FSEditListRequest(context.Context, FSEditListRequest) error
```

Comment:
The edit list and sync status stuff are completely separate.  Also, there is no registration going on at all.  The caller can ask for the complete status (presumably on startup), and can listen on the notification channel for the periodic updates, which will happen regardless of whether anyone asked for the complete status or not.